### ISSUE TITLE
svm 中的 gama 和 C

### ISSUE THREAD

**Issue by bringtree** [bringtree/everydayCommond#93]
C是错误类别的权值。我也觉得怪怪的
loss = C*badcase + L2 
gamma 是边界划分的时候  判断边界划分的 点比如（xx距离内的点来决定这条边的划分）
